tests/tests/test_security.py

```diff
@@ -78,27 +75,11 @@ def test_token_token_expiration(self, standard_setup_with_short_lived_token):
             and that deployments are still recieved by the client
         """
 
-        mender_clients = standard_setup_with_short_lived_token.get_mender_clients()
-
-        if not env.host_string:
-            execute(self.test_token_token_expiration,
-                    standard_setup_with_short_lived_token,
-                    hosts=mender_clients)
-            return
-
-        timeout_time = int(time.time()) + 60
-        while int(time.time()) < timeout_time:
-            with quiet():
-                output = run("journalctl -u mender-client -l --no-pager | grep \"received new authorization data\"")
-                time.sleep(1)
-
-            if output.return_code == 0:
-                logger.info("mender logs indicate new authorization data available")
-                break
-
-        if timeout_time <= int(time.time()):
-            pytest.fail("timed out waiting for download retries")
-
+        mender_device = standard_setup_with_short_lived_token.device
+        mender_device.run(
+            'journalctl -u mender-client -l --no-pager | grep "received new authorization data"',
```

Comment:
Nitpick: 
`journalctl --grep="received new authorization data"` maybe?
That means the `-l` and `--no-pager` options should be no longer needed either.

Reply:
I'll try this

Reply:
It seems like the journalctl compiled in yocto does not support this:
```
[172.21.0.5:8822] run: journalctl -u mender-client --grep="received new authorization data"
[172.21.0.5:8822] err: Compiled without pattern matching support
```

The call actually returns sucess, which was very annoying...

Reply:
hah, I see. Alright. Worth a shot.